partner_auto_salesman/LICENSE/LICENSE

```diff
@@ -0,0 +1,15 @@
+    OpenERP, Open Source Management Solution
```

Comment:
I feel this file is not needed, and in any case this is not the AGPL license text.